exchange/csvfeed.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/csv"
 	"errors"
 	"fmt"
+	"github.com/rodrigo-brito/ninjabot/utils"
```

Comment:
Please, organize the imports. The project separates std libs from external libraries with a space.